app/models/competency.rb

```diff
@@ -28,4 +31,24 @@ def self.parse(spreadsheet)
     return competencies
   end
 
+  # Loop through indicator, questions, resources, and the many_to_many tables
+  # and delete them all 
+  def destroy_all
+    self.indicators.each do |i|
+      i.resources.each do |r|
```

Comment:
instead of writing this whole block each time, think a shorthand that works is:
`i.resources.each(&:destroy)`, `i.questions.each(&:destroy)`, etc.